### ISSUE TITLE
Accessibility review: Virtual Tour of Vet Library

### ISSUE THREAD

**Issue by sarah-cul** [cul-it/uls#1249]
### Site/Feature for Review
[Virtual Tour of Vet Library](https://ess.h5p.com/content/1292284516507944358)

### Accessibility Testing Checklist
- [x] Use a tool such as Siteimprove or Wave to identify any accessibility errors. Document any issues found below.
- [x] Ensure the site or feature is fully navigable using only a keyboard. Check all links, buttons, forms, and other interactive elements.
- [x] Screen reader (use VoiceOver) to ensure all content and functionality is announced correctly. Pay special attention to images, form fields, and navigation.

### List Issues Found
1.  Visible label and accessible name do not match.  (A - 2.5.3  WCAG 2.1)
`<button aria-label="Send a question to the author.">I am confused</button>`
**Solution**: aria-label should contain the button text. "I am confused: Send a question to the author."
             <img width="150" alt="image" src="https://github.com/cul-it/uls/assets/144364984/ea7bca56-af66-4566-980f-8ea982ef01fd">

2.  Page zoom is restricted  ( AA 1.4.4 Resize Text, AA 1.4.10 Reflow)
According to WCAG, visitors should be allowed to increase text size from their browser to at least 200%.
Viewport scaling, is a feature on mobile devices that lets users "pinch to zoom" or "double-tap to zoom" on specific areas of the page. This is useful for making small text easier to read without affecting the rest of the layout. Its not possible to do that on this site because of this:
  `<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">`
 **Solution**: removing **maximum-scale=1** from the meta tag, will allow users to zoom freely.
 
 3. Color contrast does not meet enhanced requirement (AAA 1.4.6 Contrast)
          "by: Educational Support Services of CVM" & "I am Confused"
            <img width="250" alt="image" src="https://github.com/cul-it/uls/assets/144364984/0b7477c8-9b5a-444b-9dea-093a56b45a0a">

4. Text is clipped when resized (AA 1.4.4 Resize Text)
"I am Confused" will be clipped if resized.
 This might be due to the class **.h5p-content ul.h5p-actions** which has a fixed height of 25px. 
 
 5. I think the little circle icon shown for each section of the tour indicates that the user has toured it, but a screen reader would not have this information (as far as I can tell). Could an aria-label be used to help convey this information? 
` <div class="h5p-interactive-book-navigation-chapter-progress icon-chapter-done"></div>`
            <img width="200" alt="image" src="https://github.com/cul-it/uls/assets/144364984/2d50e0e2-2c36-4363-8e9e-ea65f3298740">
